pkg/scheduler/algorithm/predicates/csi_volume_predicate.go

```diff
@@ -55,12 +58,12 @@ type CSIMaxVolumeLimitChecker struct {
 
 // NewCSIMaxVolumeLimitPredicate returns a predicate for counting CSI volumes
 func NewCSIMaxVolumeLimitPredicate(
-	csiNodeInfo CSINodeInfo, pvInfo PersistentVolumeInfo, pvcInfo PersistentVolumeClaimInfo, scInfo StorageClassInfo) FitPredicate {
+	csiNodeInfo v1beta1storagelisters.CSINodeLister, pvInfo corelisters.PersistentVolumeLister, pvcInfo corelisters.PersistentVolumeClaimLister, scInfo storagelisters.StorageClassLister) FitPredicate {
```

Comment:
Parameter names also need to be consistent *Lister.

Reply:
+1

Reply:
Done.